.buildkite/scripts/test-with-integrations.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+
+set -euo pipefail
+
+WORKSPACE="$(pwd)"
+
+TMP_FOLDER_TEMPLATE_BASE="tmp.${GITHUB_PR_BASE_REPO}"
+TMP_FOLDER_TEMPLATE="${TMP_FOLDER_TEMPLATE_BASE}.XXXXXXXXX"
+
+cleanup() {
+    echo "Deleting temporal files..."
+    cd ${WORKSPACE}
+    rm -rf "${TMP_FOLDER_TEMPLATE_BASE}.*"
+    echo "Done."
+}
+
+trap cleanup EXIT
+source .buildkite/scripts/install_deps.sh
+
+add_bin_path
+
+echo "--- install gh cli"
+with_github_cli
+
+echo "--- install jq"
+with_jq
+
+
+INTEGRATIONS_SOURCE_BRANCH=main
+INTEGRATIONS_GITHUB_OWNER=elastic
+INTEGRATIONS_GITHUB_REPO_NAME=integrations
+INTEGRATIONS_PR_BRANCH="test-${GITHUB_PR_BASE_REPO}-pr-${BUILDKITE_PULL_REQUEST}"
+INTEGRATIONS_PR_TITLE="Test ${GITHUB_PR_BASE_REPO} - DO NOT MERGE"
```

Comment:
Btw, could we also include the pull request to be tested or something in the title so not all the created PRs are titled the same?